### DISCUSSION TITLE
Issue with Missing Activities in Feed on Home Page

### DISCUSSION THREAD

**Discussion by HosseinShahverdi** [community/community#146854] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello all

I hope you're doing well! I'm reaching out because I've noticed an issue with my GitHub homepage feed, and I could really use your help to figure it out.  

In the past, my feed used to show a mix of activities like repositories starred by people I follow, personalized recommendations, and other updates. But now, it only shows when someone follows me, and nothing else seems to show up.  

Here’s what I’ve tried so far to fix it:  
- Made sure all feed filters (announcements, sponsors, etc.) are turned on.  
- Cleared my browser cache.  
- Double-checked my notifications and activity settings to ensure everything's set up correctly.  

Unfortunately, none of these steps have worked.  

The feed is really important for me to stay updated and connected, so I’d be super grateful if you could help me figure out what’s going on. If you need any more details or info from my end, just let me know, and I’ll provide it right away.  

Thanks so much for taking the time to look into this—I appreciate it! I am looking forward to hearing from you.

**Comment by harmeetsingh11** [comment, marked as answer]
Hi @HosseinShahverdi 

It seems like what you're experiencing is related to an update in how the GitHub feed operates. The feed style has changed as of **September 2023**. The older feed style, which showed activities like repositories starred by people you follow, personalized recommendations, etc., has been updated. 

To learn more about these changes, check out this GitHub blog post that outlines the updates: https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/.

![github_blog](https://github.com/user-attachments/assets/2bc56c01-18d7-4126-89cb-c2e858c46336)

This blog outlines the new approach and how the feed now prioritizes activities differently.

### Steps to Troubleshoot and Adjust Your GitHub Feed::

1. **Reset Feed Filters:**

- Go to your GitHub homepage feed.
- Click on the filter button (usually found on the top right of the feed area).
- Press the "Reset to Default" button. This might restore your feed to its original settings.
![github_ss](https://github.com/user-attachments/assets/e7f92c56-e7cd-4113-a709-f91411c46053)

2. **Get updates from your starred repositories**

Check the option "Include events from starred repositories". This will allow your feed to display activities and events related to repositories you've starred, such as new issues, pull requests, or updates.

![github_ss1](https://github.com/user-attachments/assets/890651e3-6442-469f-8da8-db0746b862d0)

3. **Browser and Cache:**

- If resetting doesn’t work, try accessing GitHub from a different browser or incognito mode. You’ve cleared the cache, which is great, but browser extensions may still cause interference. 

Let me know if you need further help. Have a good day:)

**Reply by HosseinShahverdi** [reply]
Hello Harmeet, 

Thank you very much for your prompt response and kind assistance. I was able to restore exactly what I was looking for. I truly appreciate your help!

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @HosseinShahverdi !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Feed category. I've gone ahead and moved it for you. Good luck!

**Comment by KieranP** [comment]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, I feel free to star the repo :-)

**Reply by shinybrightstar** [reply]
Hi @KieranP , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
